AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomGPU_AreaLightScreenshotTest.py

```diff
@@ -180,13 +201,20 @@ def AtomGPU_LightComponent_AreaLightScreenshotsMatchGoldenImages():
                 AtomComponentProperties.light('Light type')) == LIGHT_TYPES['sphere'])
 
         # 14. Enter game mode and take a screenshot then exit game mode.
-        enter_exit_game_mode_take_screenshot("AreaLight_5.ppm", Tests.enter_game_mode, Tests.exit_game_mode)
+        enter_exit_game_mode_take_screenshot(screenshot_names[4], Tests.enter_game_mode, Tests.exit_game_mode)
 
         # 15. Delete the Area Light entity.
         area_light_entity.delete()
         Report.result(Tests.area_light_entity_deleted, not area_light_entity.exists())
 
-        # 16. Look for errors.
+        # 16. Compare screenshots to golden images.
+        for screenshot_name, screenshot_threshold in zip(screenshot_names, screenshot_thresholds):
+            compResult = compare_screenshot_to_golden_image(FOLDER_PATH, screenshot_name, screenshot_name)
+            Report.result(("Screenshot comparison succeeded.", "Screenshot comparison failed."), compResult.result_code == azlmbr.utils.ImageDiffResultCode_Success)
```

Comment:
include screenshot_name in your tuple strings here for failure so we can tell which one failed.
("Screenshot succeeded", f"Screenshot {screenshot_name} failed comparison")
also this is actually not comparison failed so much as comparison was not possible because result_code != azlmbr.utils.ImageDiffResultCode_Success
looks like other than success here is AZ_Error condition for mostly unsupported image conditions.